### PR TITLE
Add lint job to the pipeline

### PR DESCRIPTION
**Pull request by JosephMart**

### CODE REVIEW

File: src/handlers/BaseHandler.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import got from "got";
 
 export type GroupMeInfo = {
-  attachments: Array<any>;
+  attachments: Array<string>;
```
Comment (tristan957) [inline src/handlers/BaseHandler.ts]: string[] > Array<string> ;)

Comment (JosephMart) [reply src/handlers/BaseHandler.ts]: lmao aight

File: src/handlers/HeyHandler.test.ts [hunk 2]
```diff
@@ -13,7 +13,7 @@ const DefaultGroupMeInfo = {
   source_guid: "GUID",
   system: false,
   text: "Hello world ☃☃",
-  user_id: "1234567890"
+  user_id: "1234567890",
```
Comment (JosephMart) [inline src/handlers/HeyHandler.test.ts]: how i fix this ` 16:3  error  Identifier 'user_id' is not in camel case      @typescript-eslint/camelcase`

File: .prettierrc.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  semi: true,
+  trailingComma: "all",
+  printWidth: 120,
+  tabWidth: 2,
+  arrowParens: "avoid",
```
Comment (tristan957) [inline .prettierrc.js:6]: oof

Comment (JosephMart) [reply .prettierrc.js:6]: what you want

Comment (tristan957) [reply .prettierrc.js:6]: Doesn't really matter. I can lean either way on arrow parens. The prettier 2 rel notes had good reasons, but in the end it doesnt even matter

**Pull request merged**